src/pykeen/nn/weighting.py

```diff
@@ -70,8 +70,6 @@ class EdgeWeighting(nn.Module):
 
     def __init__(self, **kwargs):
         # stub init to enable arbitrary arguments in subclasses
-        if kwargs:
-            raise TypeError(f"keyword arguments should not be passed to {self.__class__}.__init__()")
```

Comment:
we could emit a debug log here.

Reply:
Nope my idea was wrong. The debug that was there before wouldn’t be informative either because it doesn’t know if the init has been subclasses and overridden